### DISCUSSION TITLE
Power monitoring

### DISCUSSION THREAD

**Discussion by ChRav** [Ralim/IronOS#1179] [Ideas]
It would be nice to have a little power reading on the soldering mode like the original one from the ts80/p that shows the real time power the iron is using.

**Comment by paulfertser** [comment]
On Thu, Jan 13, 2022 at 09:44:20AM -0800, ChRav wrote:
> It would be nice to have a little power reading on the soldering mode like the
> original one from the ts80/p that shows the real time power the iron is using.

It shows current power in detailed soldering screen. Is that not what
you want?

-- 
Be free, use free (http://www.gnu.org/philosophy/free-sw.html) software!
***@***.***

**Reply by ChRav** [reply]
oh, I didn't even see it, I'll ceck it out
thanks

**Reply by Ralim** [reply]
Yep, detailed mode shows an estimate similar to how the miniware firmware does :) that is your best bet
